README.md

```diff
@@ -406,6 +407,16 @@ grafana:
 
 webui:
   url: "" # WebUI URL, if not empty, WebUI output is enabled
+
+yandex:
+  # accesskeyid: "" # yandex access key
+  # secretaccesskey: "" # yandex secret access key
+  # region : "" # yandex region (default: ru-central-1)
+  s3:
+    #endpoint: "" # yandex s3 endpoint (default: ru-central-1)
+    # bucket: "falcosidekick" # Yandex S3, bucket name
+    # prefix : "" # name of prefix, keys will have format: s3://<bucket>/<prefix>/YYYY-MM-DD/YYYY-MM-DDTHH:mm:ss.s+01:00.json
+    # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|erro
```

Comment:
OK for me, can you also add these values into env vars section of README. please